### PR TITLE
CSI-3122 OLM directories for 1.5.1

### PR DESCRIPTION
**Pull request by zingero**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 packageName: ibm-block-csi-operator-community
 channels:
 - name: stable
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml:3]: as I already [wrote](https://github.com/IBM/ibm-block-csi-operator/pull/186/files#r659147162) (in a very similar PR) - I don't think this is the correct channel for 1.5.1

Comment (zingero) [reply deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml:3]: @ArbelNathan some information about OLM channels:
https://olm.operatorframework.io/docs/best-practices/channel-naming/

Comment (ArbelNathan) [reply deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml:3]: Is it our decision?

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml:3]: yes, we need an additional channel

Comment (ArbelNathan) [reply deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml:3]: added new channel

File: deploy/olm-catalog/ibm-block-csi-operator/1.5.1/bundle-1.5.1.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.6,v4.7"
+LABEL com.redhat.delivery.operator.bundle=true
+LABEL com.redhat.delivery.backport=true
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator/1.5.1/bundle-1.5.1.Dockerfile:14]: in our certified versions, I think @barakda decided to remove this label.
so we should consider to do the same for community (at least as of 1.6.0, not sure about 1.5.1 though)

Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/1.5.1/bundle-1.5.1.Dockerfile:14]: I see it was already done in the work in progress for community 1.6.0.
let's leave it as is here

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by zingero — COMMENTED** [review]
I can't approve my PR, but I approve the second commit

**Review by ArbelNathan — APPROVED** [review]

**Pull request merged**